collect_app/src/main/java/org/odk/collect/android/widgets/SelectMultiWidget.java

```diff
@@ -140,6 +137,13 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
 
     protected void createLayout() {
         if (items != null) {
+
```

Comment:
To keep things consistent, would be good to remove this newline. In this project, sometimes linebreaks are used before blocks to create visual separation but not directly inside blocks.

Reply:
Done